var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -114,7 +114,7 @@ class Petsc(Package):
     depends_on('mpi', when='+mpi')
 
     # Build dependencies
-    depends_on('python@2.6:2.8', type='build')
```

Comment:
I've asked the authors.  Until then, I'm assuming it's Python2 only.

Reply:
so the intention is to avoid using Spack's python and hope the user has python 2 in path? 

p.s. I thought there were some PRs merged on conretization of build dependencies separately (@adamjstewart @alalazo ?) that might help here to have 2 different version of python in the graph.

p.p.s. you should also adjust slepc then

Reply:
@davydden Yes that is the intention.  Whatever concretization fixes were merged, they don't solve this problem.  If you try concretizing a software stack that involves `python@3` and `petsc`, concretization will fail.  This PR is never going to be merged, it is simply a resource for people to get their Python3 stuff working.

> p.p.s. you should also adjust slepc then

Since I don't use `slepc`, I'm not going to bother.

Reply:
> This PR is never going to be merged, it is simply a resource for people to get their Python3 stuff working.

ah, i see, i missed that.

Reply:
> which still hasn't been merged

☹️